indico/htdocs/sass/modules/abstracts/_abstract_page.scss

```diff
@@ -218,9 +231,24 @@
 }
 
 .abstract-comment-badges {
-    align-self: flex-end;
+    margin-right: auto;
+    align-self: baseline;
     color: $gray;
-    margin-left: 1em;
+}
+
+.i-timeline-item-metadata {
```

Comment:
Or maybe not, because there's a `div.flex` in between the container and the items.